Formula/d/dpcmd.rb

```diff
@@ -27,6 +27,7 @@ class Dpcmd < Formula
   def install
     system "make"
     bin.install "dpcmd"
+    include.install "ChipInfoDb.dedicfg"
```

Comment:
Maybe I'm misreading https://github.com/DediProgSW/SF100Linux/pull/83, but it seems like this should be where we're installing this?
```suggestion
    (share/"DediProg").install "ChipInfoDb.dedicfg"
```

Reply:
Oh sorry, that is true. I used another patch for the one in my tap.

Reply:
Ok that was fixed. Thanks for pointing that out. I have tested the build fresh again and it did work as intended.

```
> HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source dpcmd
==> Auto-updating Homebrew...
Adjust how often this is run with HOMEBREW_AUTO_UPDATE_SECS or disable with
HOMEBREW_NO_AUTO_UPDATE. Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
Warning: No remote 'origin' in /opt/homebrew/Library/Taps/superjeng1/homebrew-tap, skipping update!
==> You have HOMEBREW_NO_INSTALL_FROM_API set
Homebrew >=4.1.0 is dramatically faster and less error-prone when installing
from the JSON API. Please consider unsetting HOMEBREW_NO_INSTALL_FROM_API.
This message will only be printed once.


==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> Tapping homebrew/cask
Cloning into '/opt/homebrew/Library/Taps/homebrew/homebrew-cask'...
remote: Enumerating objects: 1112166, done.
remote: Counting objects: 100% (168607/168607), done.
remote: Compressing objects: 100% (561/561), done.
remote: Total 1112166 (delta 168148), reused 168046 (delta 168046), pack-reused 943559 (from 1)
Receiving objects: 100% (1112166/1112166), 322.21 MiB | 6.95 MiB/s, done.
Resolving deltas: 100% (831065/831065), done.
Tapped 2 commands and 7169 casks (7,240 files, 358.5MB).

You have 56 outdated formulae and 1 outdated cask installed.

==> Fetching dpcmd
==> Downloading https://github.com/DediProgSW/SF100Linux/archive/refs/tags/V1.14.21,x.tar.gz
==> Downloading from https://codeload.github.com/DediProgSW/SF100Linux/tar.gz/refs/tags/V1.14.21%2Cx
######################################################################################################################################################################################################################################################### 100.0%
==> make
🍺  /opt/homebrew/Cellar/dpcmd/1.14.21: 7 files, 846.6KB, built in 1 second
==> Running `brew cleanup dpcmd`...
Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
Removing: /Users/willyhsiao/Library/Caches/Homebrew/dpcmd_bottle_manifest--1.14.20... (12.4KB)
Removing: /Users/willyhsiao/Library/Caches/Homebrew/dpcmd--1.14.20... (56.1KB)
Removing: /Users/willyhsiao/Library/Caches/Homebrew/dpcmd--1.14.20.tar.gz... (110.4KB)

> dpcmd --list

DpCmd macOS 1.14.21.12 Engine Version:
Last Built on Nov 14 2024


Alverstone-128M(Top)		by Micron(Numonyx)
Alverstone-128M(Bottom)		by Micron(Numonyx)
P5Q-128M		by Micron(Numonyx)
M25P05A		by Micron(Numonyx)
M25P05VP		by Micron(Numonyx)
M25P10A		by Micron(Numonyx)
[.....]
```